src/index.html

```diff
@@ -9,10 +9,57 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="style.scss"
+    />
```

Comment:
The `href` attribute in the link tag for stylesheets should point to a `.css` file, not a `.scss` file. SCSS files are preprocessed and should be compiled into CSS. Make sure to link to the compiled CSS file instead.